src/Core/Header.php

```diff
@@ -87,8 +87,13 @@ public static function setupHeader($assets = [], $echoOutput = true)
         $output = "\n<meta charset=\"utf-8\" />\n";
         // Makes only compatible with MS Edge
         $output .= "<meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\" />\n";
-        // BS4 required tag
-        $output .= "<meta name=\"viewport\" content=\"width=device-width, initial-scale=1, shrink-to-fit=no\" />\n";
```

Comment:
BS takes care of responsive behavior so suggest not to interfere with their calculation. If you noticed any issues, there is a good chance that it was because script and/or library modified the same --or-- additional form-factor specific tags (sm,lg etc) were not included.

BS5 also changes this to 
`<meta name="viewport" content="width=device-width, initial-scale=1">`

Reply:
We have been removing any jquery-ui throughout core so not a good idea to introduce in portal.

Reply:
@sjpadgett now I remember that is why I stopped using jquery.